src/main/java/use_case/club_update_desc/ClubUpdateDescOutputBoundary.java

```diff
@@ -0,0 +1,19 @@
+package use_case.club_update_desc;
+
+/**
+ * The output boundary for the Club Update Description use case.
+ */
+public interface ClubUpdateDescOutputBoundary {
+
+    /**
+     * Prepares the message for the Club Home View. Also updates the state description to match the new one.
+     * @param outputData the outputData passed to the view
+     */
+    void prepareNewDescMessage(ClubUpdateDescOutputData outputData);
+
```

Comment:
I see no problem with this, though it might be confusing, since for all other use cases, we use prepareSuccessView.

Reply:
Yeah, I think this is a valid concern. I'll change it to something more clear